src/libs/Navigation/Navigation.js

```diff
@@ -3,10 +3,18 @@ import React from 'react';
 import {StackActions, DrawerActions} from '@react-navigation/native';
 import {getIsDrawerOpenFromState} from '@react-navigation/drawer';
 
+import Onyx from 'react-native-onyx';
 import linkTo from './linkTo';
 import ROUTES from '../../ROUTES';
 import SCREENS from '../../SCREENS';
 import CustomActions from './CustomActions';
+import ONYXKEYS from '../../ONYXKEYS';
+
+let isLoggedIn = false;
+Onyx.connect({
+    key: ONYXKEYS.SESSION,
+    callback: val => isLoggedIn = !!val.authToken,
```

Comment:
If there is no `session` value at all then we will get `null` here and try to access `null['authToken']` which threw an error for me on dev. The `!!` thing is pretty uncommon to see in our codebase so probably shouldn't be used.

Try something like this maybe instead

```js
Boolean(val && val.authToken);
```